ReactAndroid/build.gradle

```diff
@@ -92,8 +94,14 @@ task prepareFolly(dependsOn: dependenciesPath ? [] : [downloadFolly], type: Copy
 }
 
 task prepareHermes() {
-    def hermesAAR = file("$projectDir/../node_modules/hermes-engine/android/hermes-debug.aar")
-    if (!hermesAAR.exists()) {
+    def hermesAAR = null
+    def hermesPackagePath = findNodeModulePath(projectDir, "hermes-engine")
+    if (hermesPackagePath) {
+        hermesAAR = file("$hermesPackagePath/android/hermes-debug.aar")
+        if (!hermesAAR.exists()) {
+            throw new GradleScriptException("The hermes-engine npm package is missing \"android/hermes-debug.aar\"")
+        }
+    } else {
```

Comment:
I don’t think we need this else case anymore since it should get picked by the node algorithm.